src/plus/startWork/startWork.ts

```diff
@@ -518,6 +527,16 @@ export class StartWorkCommand extends QuickCommand<State> {
 		void openUrl(item.item.issue.url);
 	}
 
+	private sendItemActionTelemetry(action: 'soft-open', item: StartWorkItem, context: Context) {
+		this.container.telemetry.sendEvent('startWork/issue/action', {
+			...context.telemetryContext!,
+			action: action,
+			'item.id': getStartWorkItemIdHash(item),
+			'item.type': item.item.issue.type,
+			'item.provider': item.item.issue.provider.id,
```

Comment:
We should send at least the same properties of the chosen item as we do in `startWork/issue/chosen`. If there are others we can send (that don't include private information), feel free to expand both. Perhaps we should create a function for forming the single-item telemetry payload and use it in both places.

Reply:
fixed